main.js

```diff
@@ -1,56 +1,68 @@
 const myTodoForm = document.getElementById("myTodoForm");
 const userInput = document.getElementById("userInput");
 const myList = document.getElementById("myList");
+const addButton = document.getElementById("add-btn");
+addButton.addEventListener("click", addTodo);
 
-//checking for todos in `localStorage` from a previous session
-//and rendering them to the DOM
-function displayTodo() {
-  myList.innerHTML = localStorage.getItem("todos");
-}
-displayTodo();
-
-//preventing the default form submission behavior
 myTodoForm.addEventListener("submit", function (e) {
   e.preventDefault();
 });
 
-//call 'addTodo' function
-function addTodo() {}
-
-//defining addTodo function, capturing data
 function addTodo() {
   if (userInput.value === "") {
-    alert("You must write something!");
+    const modal = document.createElement("div");
```

Comment:
You can make it simpler by adding a string with layout to the created div using innerHTML. This will simplify understanding of what you are rendering and bring you closer to understanding the concepts embedded in modern frameworks like React and Angular.